adapters/repos/db/helpers/helpers.go

```diff
@@ -20,17 +20,10 @@ import (
 var (
 	ObjectsBucket              = []byte("objects")
 	ObjectsBucketLSM           = "objects"
-	CompressedObjectsBucketLSM = "compressed_objects"
+	CompressedVectorsBucketLSM = "vectors"
```

Comment:
can it be renamed to `compressed_vectors`? flat index can store uncompressed vectors as well, there will be `VectorsBucketLSM` needed.